package.json

```diff
@@ -80,6 +80,8 @@
     "prosemirror-schema-basic": "^1.1.2",
     "prosemirror-schema-list": "^1.1.6",
     "prosemirror-state": "^1.3.4",
+    "prosemirror-tables": "^1.2.5",
```

Comment:
Can't see where this is being used, is this a necessary dependency?

Reply:
`prosemirror-utils` 0.9.6 requires `prosemirror-tables`. I have now upgraded `prosemirror-utils` to 1.0.0-0, which doesn't need the dependency anymore